django_elastic_migrations/utils/es_utils.py

```diff
@@ -1,11 +1,10 @@
-from __future__ import print_function
 import hashlib
 import json
 
 
 def get_index_hash_and_json(index):
     spec = index.to_dict()
-    json_str = json.dumps(spec, sort_keys=True).encode('utf-8')
+    json_str = json.dumps(spec, sort_keys=True)
     md5_hash = hashlib.md5()
-    md5_hash.update(json_str)
+    md5_hash.update(json_str.encode('utf-8'))
```

Comment:
In Django 2.0 release notes, they mention they ["Removed support for bytestrings in some places"](https://docs.djangoproject.com/en/2.1/releases/2.0/#removed-support-for-bytestrings-in-some-places). One of those places is saving text to a standard `TextField`, where starting in 2.0 if you save a bytestring it looks like `b'my bytestring'` in the database; you need to supply it a normal `str` to get the expected behavior.

At the same time, md5_hash requires a bytestring, so this moves the `.encode('utf-8')` to just that function, but returns the `str` that comes out of `json.dumps`.